applications/portal/frontend/src/components/viewers/detailsViewer/DetailsViewer.tsx

```diff
@@ -467,7 +476,7 @@ const DetailsViewer = (props: {
                                                     {pdfNameOnFile(file.name)}
                                                 </MenuItem>
                                                 {
-                                                    selectedIndex === index && <ListItemIcon>
+                                                    selectedIndex === index && !searchQuery && <ListItemIcon>
```

Comment:
Searching is changing the selected item, but it shouldn't:

https://github.com/MetaCell/salk-interactive-atlas/assets/19196034/74a22ed1-68ce-4c7d-bf06-1fa21a2155cc